zebra-rpc/src/methods/types/z_validate_address.rs

```diff
@@ -70,3 +71,75 @@ impl From<&Address> for ZValidateAddressType {
         }
     }
 }
+
+/// Validates a Zcash address against a network and returns a structured response.
```

Comment:
```suggestion
/// Checks if a zcash address of type P2PKH, P2SH, TEX, SAPLING or UNIFIED is valid.
/// Returns information about the given address if valid.
```

The reference pull request used the same description as the RPC method in the created public function.

https://github.com/dorianvp/zebra/blob/8cf4197291f8418764c383d6dc750d396460ae21/zebra-rpc/src/methods/types/validate_address.rs#L37-L38

We can do the same here with https://github.com/ZcashFoundation/zebra/blob/23a483da3a7133e6448f9966ec4be90567f20fe4/zebra-rpc/src/methods.rs#L548-549 for consistency.

Reply:
Yeap, this is fine. Thanks!